### DISCUSSION TITLE
DVD menu, cannot play episode 1, 2 or...4. not clicking

### DISCUSSION THREAD

**Discussion by rows-sherpa-picnic3400** [smplayer-dev/smplayer#1097] [Q&A] [answered]
Hello, I tried toggling on mplayer setting, I looked for a Drives setting, but checkbox setting for menu is not there. DVD plays episode 1, and auto plays episode 2. Somehow, if I let it play. using .iso file from Archive.org. 
https://archive.org/details/rune-soldier-dvd
I would like to be able to click the DVD menu, how to get this to work? So I can figure out which episodes I've not seen.
Also tried looking in Mouse settings, but no "activation" for Left click. I tried tab key, but DVD menu still inactive. Thanks.

**Comment by smplayer-dev** [comment, marked as answer]
The DVD menu support was removed, so clicking wouldn't work. You should able to select another title using the option Browse -> Title.

**Comment by rows-sherpa-picnic3400** [comment]
ok thanks. Browse title, that is working.
